benchmarks/comparisons/maskers.py

```diff
@@ -27,3 +27,40 @@ def peakmem_compare_mask(self, implementation, loader):
         """Peak memory of loading and then masking."""
         mask, img = load(loader)
         apply_mask(mask, img, implementation)
+
+
+class CompareParallelMask(Benchmark):
+    """
+    Comparison between the performance of applying several masks to an
+    image by parallelizing nilearn masker objects vs. using numpy
+    """
+
+    timeout = 180
```

Comment:
`nilearn, implementation - nibabel, loader` combination was taking too long and timing out. Probably should decrease the size of the image.